spacy/tests/pipeline/test_entity_linker.py

```diff
@@ -1219,3 +1219,47 @@ def test_span_maker_forward_with_empty():
     # just to get a model
     span_maker = build_span_maker()
     span_maker([doc1, doc2], False)
+
+
+def test_sentence_crossing_ents():
```

Comment:
Could you incorporate this test into `test_issue7065_b`? It's basically the same except for which entities to define, so `entities` and `links` can be parametrized.